packages/ozone-api/ozone-api-edit-video/src/ozone-api-edit-video.ts

```diff
@@ -160,14 +145,10 @@ export class OzoneApiEditVideo {
 		if (OzoneFormat.type[type]) {
 			identifier = OzoneFormat.type[type]
 		}
-		const url = `${getDefaultClient().config.ozoneURL}/rest/v3/filetype/identifier/${identifier}`
-		// No senses to do this work by chance on flowr
-		// should be the format of the video under edition
-
-		const request = new OzoneAPIRequest()
-		request.url = url
-
-		return ((await request.sendRequest()).response) as OzoneType.FileType
+		const fileTypeCache = await getDefaultClient().fileTypeClient().getFileTypeCache()
+		const file = fileTypeCache.findByIdentifier(identifier)
```

Comment:
shouldn't this be `await`ed ?

Reply:
no FileTypeCache does not return promises